tools/disasm/functions.txt

```diff
@@ -112,9 +112,9 @@
     0x800849A0:("Load_Relocate",),
     0x80084C0C:("Load_LoadOverlay",),
     0x80084CD0:("Load_AllocateAndLoad",),
-    0x80084DB0:("Load2_Relocate",),
-    0x8008501C:("Load2_LoadOverlay",),
-    0x800850C8:("Load2_AllocateAndLoad",),
+    0x80084DB0:("Overlay_Relocate",),
+    0x8008501C:("Overlay_Load",),
+    0x800850C8:("Overlay_AllocateAndLoad",),
```

Comment:
While I can agree on it being weird having completely different prefixes (though since one is completely unused I don't see it as important), I definitely would prefer consistency with OoT who use these function names. Though happy to take naming suggestions for the unused ones, maybe `UnusedOverlay_`?

Reply:
We have been avoiding to name anything "unused", since the fact the system is unused is not a fundamental part of the system.
Since the system is unused, I guess we could have a name a bit longer too

Reply:
> I don't see how `Fragment` is more descriptive than `Load` or `Overlay`, I think the rename is fine

Because it's what the chunks that are being loaded are actually called (for some reason). "Load" could load anything, "Overlay" isn't really correct either since `code` is just as much an overlay in that sense.

Since this system (and variants thereof) is used in other games, it makes sense to use the common terminology.

Reply:
So to be clear would you suggest going with `Fragment` and `Fragment2` for the system names, i.e.,
`Fragment_Relocate/Fragment2_Relocate`?

Reply:
So with that I don't mind using Fragment for the unused system (loadfragment.c), though I would still prefer `Overlay` for the used ones again for consistency with OoT.

Reply:
Then at least add a note on what the known original name of the system is in the file header, and note in the file comment that it's specifically for the `fragment` type of overlays that have the .ovl section attached to the end, rather than any old overlay.

Reply:
So I added a few comments to help clarify. Let me know if that isn't enough.